src/Duke.txt

```diff
@@ -0,0 +1,3 @@
+T | 1 |  read
```

Comment:
Is it really necessary for this to be shown to the user? 

I might put this in my .gitignore